release.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+function usage() {
+    echo
+    echo "Changes the image.*.yaml files and regenerates the build directories"
+    echo 'Commits the changes on a branch release_$RELEASE, ready for a push'
+    echo
+    echo "Usage: release.sh [options]"
+    echo
+    echo "required arguments"
+    echo
+    echo "  -r RELEASE          The oshinko release number, like v0.4.4"
+    echo
+    echo "optional arguments:"
+    echo
+    echo "  -o OSHINKO_VERSION  If not specified, the oshinko version will be set to RELEASE"
+    echo
+    echo "  -s SPARK_VERSION    The spark version, like 2.2.1"
+    echo "                      This value is used to download the spark distribution, and for the"
+    echo "                      scala image it's used to determine the openshift-spark base image"
+    echo
+    echo "  -l SCALA_VERSION    The scala version, like 2.11.8. Applies to the scala image"
+    echo
+    echo "  -t SBT_VERSION      The sbt version, like 0.13.13. Applies to the scala image"
+    echo
+    echo "  -h                  Show this message"
+}
+
+if [ "$#" -lt 1 ]; then
+    usage
+    exit 1
+fi
+
+# Set a default for the hadoop version
+HVER=2.7
```

Comment:
With no way to override (and we probably don't really need one) this seems more than just a default